### PR TITLE
baseplatform: Add section on creating a manifest

### PR DESCRIPTION
**Pull request by jonte**

### CODE REVIEW

File: docs/chapters/baseplatform/creating-a-manifest.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  * Directory names must be updated (for example, the TEMPLATECONF variable bust be set to match your layer structure)
+  * Image names must be updated to reflect your own image names (``core-image-pelux`` may become ``core-image-yourproject``)
+  * In general, this file can be customized to suite your project's CI needs
+* The ``Vagrantfile`` should be usable as-is, but can be modified to accommodate your project's CI and virtualized build needs
+* The ``xml`` files in the repository are the actual manifest files. The manifest files in the repository must be modified to indicatethe correct repositories. It is common to structure the manifets files in a hierachical fashion, such that a common base manifest is created, and (possibly several) specialized manifests include this base manifest. An example of this can be seen in the PELUX manifests repository, where the ``pelux-base.xml`` manifest is included in, for instance, the ``pelux-intel.xml`` and ``pelux-rpi.xml`` manifests.
```
Comment (breakreturn) [inline docs/chapters/baseplatform/creating-a-manifest.rst]: "indicatethe" -> indicate the

**Review by breakreturn — CHANGES_REQUESTED** [review]
Tiniest comment only

**Comment by breakreturn** [comment]
Feel free to merge it after adding the space between the two words i commented on.

**Comment by jonte** [comment]
Typo fixed. Thanks for the review. Merging.

**Pull request merged**